src/packages/utils.ts

```diff
@@ -74,19 +75,33 @@ export const exportToBlob = (
   });
 };
 
-export const exportToSvg = ({
+export const exportToSvg = async ({
   elements,
   appState = getDefaultAppState(),
   exportPadding,
-  metadata,
+  metadata = "",
+  embedScene,
 }: Omit<ExportOpts, "getDimensions"> & {
   exportPadding?: number;
   metadata?: string;
-}): SVGSVGElement => {
+  embedScene?: boolean;
+}): Promise<SVGSVGElement> => {
   const { elements: restoredElements, appState: restoredAppState } = restore(
     { elements, appState },
     null,
   );
+  if (embedScene) {
+    try {
+      const sceneMetaData = await (
+        await import(/* webpackChunkName: "image" */ "../../src/data/image")
+      ).encodeSvgMetadata({
+        text: serializeAsJSON(restoredElements, restoredAppState),
+      });
+      metadata += sceneMetaData;
```

Comment:
@dwelle I think we can simply remove `metadata` as it will be only useful when embedding scene data which is now possible via `embedScene` attribute, what do you think ?